AGENTS.md

```
@@ -17,9 +17,9 @@ See @\_docs/contributor-guide.md for Tech Stack, structure, commands, style, tes
 
 ## Quick Start
 
-- Dev: `bun run dev` → http://localhost:3000
-- Checks/Tests: `bun run check`, `bun run test`
-- Build/Preview: `bun run build`, `bun run serve`
+- Dev: `pnpm dev` → http://localhost:3000
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)